README.md

```diff
@@ -1,2 +1,237 @@
+          - starter
+          - swift
+          - java-microprofile
+          - nodejs
+      - url: https://github.com/kabanero-io/collections/releases/download/0.6.0-rc.1/kabanero-index.yaml
+        include:
+          - java-microprofile
+image-org:
+image-registry:
+nginx-image-name:
+```
+
+This example configuration file builds a single stack hub index file for your `default` hub. The `default` hub points to two index files, `https://github.com/appsody/stacks/releases/latest/download/incubator-index.yaml` and `https://github.com/kabanero-io/collections/releases/download/0.6.0-rc.1/kabanero-index.yaml`. The `include:` and `exclude:` options are used
+to filter from the available application stacks.
+
+2. Save your changes.
+
+You can create as many hub index files as you need from a single configuration file by following the format in the `example_config/example_repo_config.yaml` file. For example, you might want to create an index file for your production environment with one set of application stacks and one for your development or test environments where you are building apps on different underlying software levels.
+
+### Building stack hub assets
+
+Build your stack hub assets from the example configuration file by running the following command:
+
+```
+./scripts/hub_build.sh example_config/example_repo_config.yaml
+```
+
+The output from the command shows how information is fetched from each index file, which the build process consolidates into a single file. Here is some sample output:
+
+```
+Config file: example_config/example_repo_config.yaml
+ == Running pre_build.d scripts
+ ==== Running prereq_check.sh
+ == Done pre_build.d scripts
+Not retrieving or modifying any assets
+Creating consolidated index for default
+== fetching https://github.com/appsody/stacks/releases/latest/download/incubator-index.yaml
+== Adding stacks from index https://github.com/appsody/stacks/releases/latest/download/incubator-index.yaml
+==== Adding stack nodejs-loopback 0.2.2
+==== Excluding stack swift 0.2.5
+==== Excluding stack node-red 0.1.1
+==== Excluding stack python-flask 0.2.2
+==== Excluding stack starter 0.1.2
+==== Excluding stack java-microprofile 0.2.24
+==== Adding stack java-spring-boot2 0.3.26
+==== Excluding stack kitura 0.2.5
+==== Adding stack java-openliberty 0.2.2
+==== Excluding stack nodejs 0.3.4
+==== Adding stack nodejs-express 0.4.3
+== fetching https://github.com/kabanero-io/collections/releases/download/0.6.0-rc.1/kabanero-index.yaml
+== Adding stacks from index https://github.com/kabanero-io/collections/releases/download/0.6.0-rc.1/kabanero-index.yaml
+==== Excluding stack java-microprofile 0.2.25
+==== Excluding stack java-openliberty 0.2.1
+==== Excluding stack java-spring-boot2 0.3.23
+==== Excluding stack nodejs-express 0.2.8
+==== Excluding stack nodejs-loopback 0.1.8
+==== Excluding stack nodejs 0.3.1
+ == Running post_build.d scripts
+ ==== Running build_nginx.sh
+ == Done post_build.d scripts
+ ```
```

Comment:
Looking at this output has indicated a bug Ian the hub-build.sh, as the java-microprofile stack has been included from the 2nd index when it is specifically indicated as being included. I am producing. fix for this but one done this hub-build.sh output will need to be replaced or you can just change the line `==== Excluding stack java-microprofile 0.2.25` to be `Adding stack java-microprofile 0.2.25`

Reply:
I updated the line.